jenkins-job-builder/config/definitions/jjb.yml

```diff
@@ -1,6 +1,6 @@
 - job:
     name: jenkins-job-builder
-    node: (small && trusty) || master
+    node: small && trusty
```

Comment:
we can, they just take forever to provision. did you disable wheezy manually? I checked here and they still look enabled https://github.com/ceph/mita/blob/master/deploy/playbooks/roles/common/templates/prod_nodes.py.j2#L26